### PR TITLE
Mobile version navigation bar and headline section User Interface

### PR DESCRIPTION
**Pull request by kessio**
This pull request contains a navigation bar and headline section UI design code parsed from Figma. The project contains HTML and CSS files used in the creation of the UI. HTML and CSS linters has been to flag programming errors, bugs, stylistic errors and suspicious constructs. This request also contains a descriptive README file and meaningful commit messages for every step taken when creating the portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+   <link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,900&family=Plus+Jakarta+Sans:ital,wght@0,200;0,300;0,400;0,600;0,700;1,200;1,500&family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    
+        <header>
+            <nav>
+                <div class="logo">My Logo</div>
+                <a class="menu"><i class="fa fa-bars"></i></a>
+            </nav>
```
Comment (1GORDON) [inline index.html:20]: Nice job @kessio. 👏

- [ ] [Optional] You may also make the logo and hamburger clickable links. You can surround each of these with `<a>` tags.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  /* Hamburger menu */
+  color: #6070ff;
+  font-size: 20px;
+}
+
+.headline {
+  background-image: url(images/header-shapes\ mobile.png);
+  position: absolute;
+  height: 718px;
+}
```
Comment (1GORDON) [inline style.css:33]: Great work @kessio. 👏

- [ ] Please try to fix the background image. It is repeating hence making your design very different from the one provided in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md).
- Your design
back-ground

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @kessio.

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- Good commit messages  ✔️ 
- Professional Pull request ✔️
- Professional Readme ✔️
- Linters are passing ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+         If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+         <p class="socials">LET’S CONNECT</p>
+         <div class="social-icons">
+            <img class="twitter" src="images/twitter.svg" alt="twitter-icon">
+            <i class="fa fa-linkedin"></i>
+            <img class="m" src="images/m.svg" alt="m-icon">
+            <img class="github" src="images/github.svg" alt="github-icon">
+            <img class="like" src="images/like.svg" alt="like-icon">
+         </div>
```
Comment (Ismailco) [inline index.html:37]: - [ ] [OPTIONAL] The icons should be wrapped in an `ul` tag, and each icon should be a `li`, then the icon is a link sung `a` tag, kindly implement this so your project is completed

File: style.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
+* {
+  /* Styling applied to the whole page */
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+.container-homepage {
```
Comment (Ismailco) [inline style.css:7]: - [ ] [OPTIONAL] Make sure to use flex to align elements in this section

**Review by Ismailco — APPROVED** [review]
### Hi @kessio,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**